contracts/crowdsale/validation/TimedCrowdsale.sol

```diff
@@ -74,4 +81,15 @@ contract TimedCrowdsale is Crowdsale {
     function _preValidatePurchase(address beneficiary, uint256 weiAmount) internal onlyWhileOpen view {
         super._preValidatePurchase(beneficiary, weiAmount);
     }
+    
+    /**
+     * @dev Extend crowdsale
+     * @param closingTime Crowdsale closing time
+     */
+    function _extendTime(uint256 closingTime) internal {
```

Comment:
You are shadowing the `closingTime` function by having an argument with this name: either rename it to `closingTime_` (see `ERC20Migrator` for instances where we do the same thing to prevent shadowing), or choose a different name altogether: I'd go for `newClosingTime`.